src/styles/custom.css

```diff
@@ -1,5 +1,9 @@
 /** Custom Styles **/
 
+/* Ensure that <header> nav element doesn't allow overflow-y */
+.page > .header {
+	overflow-y: hidden;
+}
```

Comment:
Should this change be upstreamed to Starlight? Feels gross to have it here, but I didn't want to add a component override just to add an overflow rule.

Reply:
Ah you're totally correct. Putting the `transform` on the `<svg>` should solve the problem. Good catch!